tracer/maccess.go

```diff
@@ -26,7 +26,11 @@ func checkForMaccessPatch(coll *cebpf.CollectionSpec, maps map[string]*cebpf.Map
 
 	code, err := loadKernelCode(coll, maps, faultyFunc.Address)
 	if err != nil {
-		log.Warnf("Failed to load code for %s: %v", faultyFunc.Name, err)
+		log.Warnf("Failed to load code for %s: %v.\n"+
+			"Syscall tracepoints are not working on this system, so whether "+
+			"the kernel is patched could not be determined. "+
+			"Either use a kernel configured with syscall tracepoints, "+
+			"or upgrade to kernel version 6.4 or higher.", faultyFunc.Name, err)
```

Comment:
`loadKernelCode` can fail for various reasons. Just naming tracepoints here might be misleading. 
```suggestion
		log.Warnf("Failed to load code for %s: %v\n"+
			"Upgrading the kernel version to 6.4 or higher could resolve the issue.", faultyFunc.Name, err)
```

Reply:
I think this warning/error should be moved from here to `loadKernelCode` or even `executeSystemAnalysisBpfCode` in the approariate place. Perhaps even checking the error code to see if its likely due to kernel config?

Reply:
Is the flag `-no-kernel-version-check` to bypass this check mentioned in somewhere on this failure path?